src/test/test-execute.c

```diff
@@ -551,8 +559,14 @@ static void test_exec_capabilityambientset(Manager *m) {
                 return;
         }
 
-        test(m, "exec-capabilityambientset.service", 0, CLD_EXITED);
-        test(m, "exec-capabilityambientset-merge.service", 0, CLD_EXITED);
+        if (have_effective_cap(CAP_NET_ADMIN) <= 0 ||
+            have_effective_cap(CAP_NET_RAW) <= 0) {
```

Comment:
I think these checks are the right thing to do, but would it also make sense to replace `CAP_NET_ADMIN` and `CAP_NET_RAW` with other capabilities that are less likely to be dropped by container runtimes to minimize the probability of the test being skipped?

Reply:
OK. Do you have any proposal?

Reply:
I've skimmed over the following configs:
* https://github.com/systemd/systemd/blob/6cc7e918ffe2f6a08be2b3ea221b8dbb79e41a97/src/nspawn/nspawn.c#L153-L179
* https://github.com/moby/moby/blob/954e55b3745fe764df54e8fc19039b49bcd2f1a3/oci/defaults.go#L14-L29
* https://github.com/lxc/lxc/blob/19462d43e8e9744f08f127bfb5d5a9f08c1aafa9/src/lxc/conf.c#L209-L260
* https://github.com/rkt/rkt/blob/b429dc42790a31a2c4169d56838e1fbe671ce895/stage1/init/common/pod.go#L642-L656

and `CAP_NET_RAW` seems to be safe to use. Another one I would probably pick is `CAP_SETUID`.